SConstruct

```diff
@@ -473,8 +473,6 @@ if selected_platform in platform_list:
             sys.exit(255)
         else:
             env.Append(CPPDEFINES=['_3D_DISABLED'])
-    if env['gdscript']:
```

Comment:
The `'gdscript'` option should also be removed.

Reply:
Done.